### DISCUSSION TITLE
Most recent version in documentation

### DISCUSSION THREAD

**Discussion by brandonbiggs** [mahendrapaipuri/ceems#231] [Q&A] [answered]
Hi! 

Thanks for the work on this cool project. I'm trying to install it. I noticed that the documentation for installing from an OS package points to an older version (0.1.0) https://mahendrapaipuri.github.io/ceems/docs/installation/os-packages. 

It looks like the current version is 0.4.1. The documentation makes it clear that "CEEMS is in early development phase, thus subject to breaking changes with no guarantee of backward compatibility." so I guess I was wondering, should I be using the most recent version or should I use the older version mentioned in the docs? If I'm supported to use the most recent version, does it make sense to update the docs to point to the newest version? 

Thanks!

**Comment by mahendrapaipuri** [comment]
Hello @brandonbiggs 

Thanks a lot for your interest. 

It is always better to start with the most recent release. The docs that you pointed out is just an illustration on how to install the packages. I think it is a good idea to mention in the docs to always prefer to install most recent versions.

If you would like to install the exporter, I advise to even look into the version [`0.5.0-rc.2`](https://github.com/mahendrapaipuri/ceems/releases/tag/v0.5.0-rc.2). Although it is a pre-release, it fixes some important bugs and brings in performance benefits. I will make a 0.5.0 release very soon!

Please let me know if you encounter any issues during installation or running the stack. Would be very happy to help you!

Cheers!

**Reply by brandonbiggs** [reply]
I'll use 0.5.0-rc.2 then. Thank you!

**Comment by brandonbiggs** [comment]
@mahendrapaipuri Okay, so far things are going well! Are there default grafana dashboard configs to get the same visualizations that you have in the README?

**Reply by mahendrapaipuri** [reply]
This is something I have been asked quite a lot. I am setting up a demo of CEEMS on a cloud instance and once I open it to public, you will able to fetch dashboard models from Grafana on that instance. I am pretty close to finishing it and it should be online before the end of this week!

I will also add more "complete" dashboards in the repo soon. They both are on priority to-do list.

**Reply by brandonbiggs** [reply]
This rocks! Thank you!

**Reply by mahendrapaipuri** [reply, marked as answer]
There you go, Demo instance is live at https://ceems-demo.myaddr.tools/dashboards. More details in [README](https://github.com/mahendrapaipuri/ceems/tree/main?tab=readme-ov-file#demo). These dashboards are very basic without any access control. You will have to modify them based on your environment and needs.

You will have to setup recording rules on Prometheus as well which is discussed [here](https://github.com/mahendrapaipuri/ceems/tree/main/etc/prometheus).

Please let me know if there is something that is not clear or missing. Will help me to improve docs. 

Cheers!!
